examples/tutorials/streamwriter_basic_tutorial.py

```diff
@@ -0,0 +1,658 @@
+"""
+StreamWriter Basic Usage
+========================
+
+This tutorial shows how to use :py:class:`torchaudio.io.StreamWriter` to
+encode and save audio/video data into various formats/destinations.
+
+"""
+
+######################################################################
+#
+# .. note::
+#
+#    This tutorial requires torchaudio nightly build and FFmpeg libraries (>=4.1, <4.4).
+#
+#    To install torchaudio nightly build, please refer to
+#    https://pytorch.org/get-started/locally/ .
+#
+#    There are multiple ways to install FFmpeg libraries.
+#    If you are using Anaconda Python distribution,
+#    ``conda install 'ffmpeg<4.4'`` will install the required FFmpeg libraries.
```

Comment:
```suggestion
#    ``conda install 'ffmpeg<4.4'`` will install the required FFmpeg library.
```

Reply:
Are there multiple libraries for FFmpeg?

Reply:
Yes, FFmpeg is composed of 8 libraries. We need 7 of them.